test/test.js

```diff
@@ -19,4 +19,16 @@ describe('is-relative-path', function () {
     assert.ok(!isRelative('foo'));
     assert.ok(!isRelative('foo/bar/car/baz.js'));
   });
+
+  it('throws on empty input', function () {
+    assert.throws(function () {
+      isRelative();
+    }, function (err) { return (err instanceof TypeError); });
```

Comment:
are you considering `neat-contract` usage? if so we dont need to test `neat-contract` part of the work, otherwise I can easily add assertions for error messages as well